snap/snapcraft.yaml

```diff
@@ -230,41 +230,6 @@ parts:
           rm -rf "${CRAFT_PART_INSTALL}/${res}"
       done
 
-  opensearch-plugin-repository-gcs:
-    plugin: nil
-    after: [opensearch]
-    override-build: |
-      # update deps
-      apt-get install unzip
-      version="$(craftctl get version)"
-      archive="repository-gcs-${version}.zip"
-      url="https://artifacts.opensearch.org/releases/plugins/repository-gcs/${version}/${archive}"
-      curl -L -o "${archive}" "${url}"
-      unzip "${archive}" -d "${CRAFT_PART_INSTALL}/repository-gcs"
-      mkdir -p "${CRAFT_PART_INSTALL}/usr/share/opensearch/plugins"
-      mv "${CRAFT_PART_INSTALL}/repository-gcs" "${CRAFT_PART_INSTALL}/usr/share/opensearch/plugins"
-      # Final clean-up
-      rm "${archive}"
-
-  opensearch-plugin-repository-s3:
-    plugin: nil
-    after: [opensearch]
-    override-build: |
-      # update deps
-      apt-get install unzip
-
-      version="$(craftctl get version)"
-      archive="repository-s3-${version}.zip"
-      url="https://artifacts.opensearch.org/releases/plugins/repository-s3/${version}/${archive}"
-      curl -L -o "${archive}" "${url}"
-
-      unzip "${archive}" -d "${CRAFT_PART_INSTALL}/repository-s3"
-      mkdir -p "${CRAFT_PART_INSTALL}/usr/share/opensearch/plugins"
-      mv "${CRAFT_PART_INSTALL}/repository-s3" "${CRAFT_PART_INSTALL}/usr/share/opensearch/plugins"
-
-      # Final clean-up
-      rm "${archive}"
-
```

Comment:
Why do we remove the above?

Reply:
@juditnovak that was in the case we've downloaded the plugin jars from the internet. In this case, we have the archive + the installed files. Now, we are using the tarball in our build from source, which contains everything. Then, we do not need to do any of these anymore.